### PR TITLE
fixed email validation on offer help page

### PR DESCRIPTION
**Pull request by omarzoghayyer**
### What does this pull request aim to achieve? 💯

Added email validation to offer help page using { validateEmail } from "utils/validators";

### 🚨Before submitting this pull request🚨:

_Please do **NOT** submit this PR if you have not done the following:_

- [x] 1. I have checked that no one else is working on similar changes.
- [x] 2. I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] 3. The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] 4. This branch is rebased/merged with the **latest master**.
- [x] 5. There are no merge conflicts.
- [x] 6. There are no console warnings and errors.
- [x] 7. On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a 

project progress under "Projects".

### CODE REVIEW

File: client/src/pages/OfferHelp.js [hunk 1]
```diff
@@ -130,19 +131,23 @@ const Step2 = (props) => {
...
+    setValid(validated);
+  }, [email]);
+
+  const onChange = (event) => {
+    setEmail(event.target.value);
   };
+
   const onSubmit = () => {
-    props.update("email", email);
+    console.log("submit");
```
Comment (robinv85) [inline client/src/pages/OfferHelp.js:148]: ah yeah, now i see this one is using `props.update()` that one is passed from the page component below... we could still use that too but I'm not sure how this should work when the wizards is completed

Comment (omarzoghayyer) [reply client/src/pages/OfferHelp.js:148]: We can just have it removed for now and can fix it when the we have the full picture ready.

Comment (robinv85) [reply client/src/pages/OfferHelp.js:155]: yeah talked on slack about it and it's not clear what has to be done here, good for now

**Comment by snowiesuet** [comment]
Can we add in an error message when user tries to submit without inputting email? Also on desktop, this doesn't look right: 
image

**Comment by robinv85** [comment]
@snowiesuet the components have not been completely finsihed in terms of styling, i also didn't do that because funcitonality was priority
for the error message with a blank input, the problem is, we don't have clear requirements, i already brought this discussion up a few times on slack. We don't have a backend endpoint to save the data and we are not using it yet, so I think this is out of scope here

**Comment by snowiesuet** [comment]
> @snowiesuet the components have not been completely finsihed in terms of styling, i also didn't do that because funcitonality was priority
> for the error message with a blank input, the problem is, we don't have clear requirements, i already brought this discussion up a few times on slack. We don't have a backend endpoint to save the data and we are not using it yet, so I think this is out of scope here
> I also didn't add the "required" property to the email input, that's all there is to it

Got it!

**Pull request merged**